### PR TITLE
Nuke 5.3

### PR DESCRIPTION
**Pull request by lehinevych**

### CODE REVIEW

File: cfme/tests/configure/test_access_control.py [hunk 1]
```diff
@@ -125,11 +125,6 @@ def test_user_password_required_error_validation():
         user.create()
 
 
-@pytest.mark.meta(
-    blockers=[
-        BZ(1118040, unblock=lambda appliance_version: appliance_version < "5.3")
```
Comment (mfalesni) [inline cfme/tests/configure/test_access_control.py]: Be careful with further removing the bugs with unblock= having 5.3 in the future. This one is closed already so it does not matter.

Comment (lehinevych) [inline cfme/tests/configure/test_access_control.py]: I checked that.

**Comment by dajoRH** [comment]
Lint report for commit b093b262ffa02cdc022a0107b37fdb8931ff49a2:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by lehinevych** [comment]
The removed blockers were checked and they all have closed statuses.

**Comment by dajoRH** [comment]
Lint report for commit 45d528259fd67a7c482ee94c8f382f5cfcf699fc:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 52f38553af80353f6472aa3ca7dc7b88df90ba8a:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by mfalesni** [comment]
Code seems legit. We'll see what does the pytest say.

@psav I will create a 5.3 branch from commit preceeding this one once it is merged, just for orientation ...

**Comment by lehinevych** [comment]
@mfalesni  I will add more changes a bit later in regards of nuking 5.3

**Comment by psav** [comment]
Soudns good Thanks @mfalesni

**Comment by dajoRH** [comment]
Lint report for commit debad193561903d075a9b0ea708ceb7a15d7085d:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 356db7d3124d87cc916997482b0ec3d2b7c3ef3d:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 8c71377afb4d30cfe4511803044147fc1bba3aee:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by psav** [comment]
@mfalesni seems linter missed some things landscape did not? in configure/__init__.py there is a reference to current_version which is not used any more?

**Comment by mfalesni** [comment]
@psav Removed the comment and waiting for next round of the bot to kick in.

**Comment by dajoRH** [comment]
Lint report for commit 8c71377afb4d30cfe4511803044147fc1bba3aee:

`cfme/automate/explorer.py`:
- File lint OK :cake: :punch: :cookie:

`cfme/automate/service_dialogs.py`:
- File lint OK :cake: :punch: :cookie:

`cfme/cloud/instance.py`:
- File lint OK :cake: :punch: :cookie:

`cfme/cloud/provider.py`:
- File lint OK :cake: :punch: :cookie:

`cfme/common/__init__.py`:
- File lint OK :cake: :punch: :cookie:

`cfme/common/provider.py`:
- File lint OK :cake: :punch: :cookie:

`cfme/configure/about.py`:
- File lint OK :cake: :punch: :cookie:

`cfme/configure/configuration/__init__.py`:
- :red_circle: Line 24:1: **F401** *'current_version' imported but unused*

`cfme/configure/settings.py`:
- File lint OK :cake: :punch: :cookie:

`cfme/fixtures/storage.py`:
- File lint OK :cake: :punch: :cookie:

`cfme/infrastructure/datastore.py`:
- File lint OK :cake: :punch: :cookie:

`cfme/infrastructure/host.py`:
- File lint OK :cake: :punch: :cookie:

`cfme/infrastructure/provider.py`:
- File lint OK :cake: :punch: :cookie:

`cfme/infrastructure/pxe.py`:
- :red_circle: Line 22:1: **F401** *'version' imported but unused*

`cfme/infrastructure/virtual_machines.py`:
- File lint OK :cake: :punch: :cookie:

`cfme/intelligence/reports/menus.py`:
- File lint OK :cake: :punch: :cookie:

`cfme/provisioning.py`:
- File lint OK :cake: :punch: :cookie:

`cfme/rest/__init__.py`:
- :red_circle: Line 11:1: **F401** *'version' imported but unused*

`cfme/roles.py`:
- File lint OK :cake: :punch: :cookie:

`cfme/services/catalogs/cloud_catalog_item.py`:
- File lint OK :cake: :punch: :cookie:

`cfme/tests/automate/test_domain_priority.py`:
- File lint OK :cake: :punch: :cookie:

`cfme/tests/automate/test_method.py`:
- :red_circle: Line 5:1: **F401** *'version' imported but unused*

`cfme/tests/automate/test_namespace.py`:
- File lint OK :cake: :punch: :cookie:

`cfme/tests/automate/test_service_dialog.py`:
- File lint OK :cake: :punch: :cookie:

`cfme/tests/automate/test_smoke.py`:
- File lint OK :cake: :punch: :cookie:

`cfme/tests/automate/test_vmware_methods.py`:
- File lint OK :cake: :punch: :cookie:

`cfme/tests/cli/test_rake_evm_automate-53.py`:
- File lint OK :cake: :punch: :cookie:

`cfme/tests/cloud/test_cloud_timelines.py`:
- File lint OK :cake: :punch: :cookie:

`cfme/tests/cloud/test_instance_power_control.py`:
- File lint OK :cake: :punch: :cookie:

`cfme/tests/cloud/test_provisioning.py`:
- :red_circle: Line 13:1: **F401** *'version' imported but unused*

`cfme/tests/cloud/test_stack.py`:
- File lint OK :cake: :punch: :cookie:

`cfme/tests/configure/test_access_control.py`:
- :red_circle: Line 16:1: **F401** *'BZ' imported but unused*

`cfme/tests/control/test_bugs.py`:
- File lint OK :cake: :punch: :cookie:

`cfme/tests/infrastructure/test_config_management.py`:
- File lint OK :cake: :punch: :cookie:

`cfme/tests/infrastructure/test_datastore_analysis.py`:
- :red_circle: Line 7:1: **F401** *'version' imported but unused*

`cfme/tests/infrastructure/test_host_analysis.py`:
- :red_circle: Line 12:1: **F401** *'BZ' imported but unused*

`cfme/tests/infrastructure/test_infra_quota.py`:
- File lint OK :cake: :punch: :cookie:

`cfme/tests/infrastructure/test_provisioning_dialog.py`:
- :red_circle: Line 14:1: **F401** *'version' imported but unused*

`cfme/tests/infrastructure/test_vm_clone.py`:
- File lint OK :cake: :punch: :cookie:

`cfme/tests/integration/test_ipa_external.py`:
- File lint OK :cake: :punch: :cookie:

`cfme/tests/intelligence/reports/test_report_corresponds.py`:
- File lint OK :cake: :punch: :cookie:

`cfme/tests/services/test_add_remove_vm_to_service.py`:
- :red_circle: Line 18:1: **F401** *'version' imported but unused*

`cfme/tests/services/test_catalog_item.py`:
- File lint OK :cake: :punch: :cookie:

`cfme/tests/services/test_cloud_service_catalogs.py`:
- File lint OK :cake: :punch: :cookie:

`cfme/tests/services/test_dynamicdd_dialogelement.py`:
- File lint OK :cake: :punch: :cookie:

`cfme/tests/services/test_generic_service_catalogs.py`:
- File lint OK :cake: :punch: :cookie:

`cfme/tests/services/test_iso_service_catalogs.py`:
- File lint OK :cake: :punch: :cookie:

`cfme/tests/services/test_myservice.py`:
- :red_circle: Line 17:1: **F401** *'version' imported but unused*

`cfme/tests/services/test_orchestration_template.py`:
- File lint OK :cake: :punch: :cookie:

`cfme/tests/services/test_provision_stack.py`:
- File lint OK :cake: :punch: :cookie:

`cfme/tests/services/test_pxe_service_catalogs.py`:
- File lint OK :cake: :punch: :cookie:

`cfme/tests/services/test_request.py`:
- :red_circle: Line 14:1: **F401** *'version' imported but unused*

`cfme/tests/services/test_service_catalogs.py`:
- :red_circle: Line 18:1: **F401** *'version' imported but unused*

`cfme/tests/storage/test_basic.py`:
- File lint OK :cake: :punch: :cookie:

`cfme/tests/test_appliance.py`:
- File lint OK :cake: :punch: :cookie:

`cfme/tests/test_rest.py`:
- File lint OK :cake: :punch: :cookie:

`cfme/tests/test_soap.py`:
- :red_circle: Line 8:1: **F401** *'BZ' imported but unused*

`cfme/web_ui/__init__.py`:
- File lint OK :cake: :punch: :cookie:

`cfme/web_ui/flash.py`:
- File lint OK :cake: :punch: :cookie:

`cfme/web_ui/search.py`:
- File lint OK :cake: :punch: :cookie:

`scripts/patch_ajax_wait.py`:
- File lint OK :cake: :punch: :cookie:

`utils/appliance.py`:
- File lint OK :cake: :punch: :cookie:

`utils/testgen.py`:
- File lint OK :cake: :punch: :cookie:

`utils/version.py`:
- File lint OK :cake: :punch: :cookie:

Please, rectify these issues :smirk: .
*CFME QE Bot*

**Comment by mfalesni** [comment]
Hehe, sorry for spam here :)

**Comment by dajoRH** [comment]
Lint report for commit 8c71377afb4d30cfe4511803044147fc1bba3aee:

`cfme/configure/configuration/__init__.py`:
- :red_circle: Line 24:1: **F401** *'current_version' imported but unused*

`cfme/infrastructure/pxe.py`:
- :red_circle: Line 22:1: **F401** *'version' imported but unused*

`cfme/rest/__init__.py`:
- :red_circle: Line 11:1: **F401** *'version' imported but unused*

`cfme/tests/automate/test_method.py`:
- :red_circle: Line 5:1: **F401** *'version' imported but unused*

`cfme/tests/cloud/test_provisioning.py`:
- :red_circle: Line 13:1: **F401** *'version' imported but unused*

`cfme/tests/configure/test_access_control.py`:
- :red_circle: Line 16:1: **F401** *'BZ' imported but unused*

`cfme/tests/infrastructure/test_datastore_analysis.py`:
- :red_circle: Line 7:1: **F401** *'version' imported but unused*

`cfme/tests/infrastructure/test_host_analysis.py`:
- :red_circle: Line 12:1: **F401** *'BZ' imported but unused*

`cfme/tests/infrastructure/test_provisioning_dialog.py`:
- :red_circle: Line 14:1: **F401** *'version' imported but unused*

`cfme/tests/services/test_add_remove_vm_to_service.py`:
- :red_circle: Line 18:1: **F401** *'version' imported but unused*

`cfme/tests/services/test_myservice.py`:
- :red_circle: Line 17:1: **F401** *'version' imported but unused*

`cfme/tests/services/test_request.py`:
- :red_circle: Line 14:1: **F401** *'version' imported but unused*

`cfme/tests/services/test_service_catalogs.py`:
- :red_circle: Line 18:1: **F401** *'version' imported but unused*

`cfme/tests/test_soap.py`:
- :red_circle: Line 8:1: **F401** *'BZ' imported but unused*

Please, rectify these issues :smirk: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit fe1a3da09b997456d98d9b8a3b425efd50c23b88:

`cfme/tests/cloud/test_provisioning.py`:
- :red_circle: Line 13:1: **F401** *'version' imported but unused*

Please, rectify these issues :smirk: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit a04d269d5fe4c81f4f20c7cffbc64ce43ebf7b55:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 84ef32599528de13e387b77f96628dfc56f4c04c:

`cfme/tests/automate/test_service_dialog.py`:
- :bangbang: Line 82:3: **E227** *missing whitespace around bitwise or shift operator*
- :bangbang: Line 82:3: **E901** *SyntaxError: invalid syntax*
- :bangbang: Line 82:7: **E225** *missing whitespace around operator*
- :bangbang: Line 84:1: **E112** *expected an indented block*
- :bangbang: Line 84:3: **E225** *missing whitespace around operator*
- :bangbang: Line 84:5: **E225** *missing whitespace around operator*
- :bangbang: Line 84:7: **E225** *missing whitespace around operator*
- :bangbang: Line 85:5: **E113** *unexpected indentation*
- :bangbang: Line 87:1: **E112** *expected an indented block*
- :bangbang: Line 87:7: **E225** *missing whitespace around operator*

Please, rectify these issues :smirk: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 8a07def91863a44783809009952d0ecea71b69fb:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 7d0258c5c4c79dd9e32b081cfe97ea22452028fd:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 39baeab3942bd4e8eaba638f7eb07dc7b8f6da52:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by mfalesni** [comment]
So, I think I am merging this. You happy too @psav?

**Pull request merged**